libbeat/processors/script/javascript/javascript_test.go

```diff
@@ -139,6 +155,36 @@ func TestRun(t *testing.T) {
 		v, _ := result.GetValue("from_glob")
 		assert.Equal(t, true, v)
 	})
+
+	t.Run("without SetPaths fails", func(t *testing.T) {
```

Comment:
do you test that it works with a `SetPaths` call?

Reply:
Pushed a commit that covers it in the `with file` test